### PR TITLE
Incorporate latest load test configuration and setup parameters to install guides

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: install.rst [hunk 1]
```diff
@@ -367,7 +457,7 @@ Varying Mattermost Load Test
 Total Potential Users 
 --------------------------------------------------
 
-Setting: ``THREADCOUNT="5000"``
+Setting: ``THREADCOUNT="15000"``
```
Comment (coreyhulen) [inline install.rst:460]: These are not really used anymore.  The load test script has changed.

File: install.rst [hunk 2]
```diff
@@ -378,11 +468,10 @@ Setting: ``THREADOFFSET="0"``
 
 - Total number of unused users 
 
-
 Setup Time 
 --------------------------------------------------
 
-Setting: ``RAMPSEC="2000"``
+Setting: ``RAMPSEC="1000"``
```
Comment (coreyhulen) [inline install.rst:474]: These are not really used anymore.  The load test script has changed.

**Review by coreyhulen — APPROVED** [review]
Some old lead test stuff, but it's probably ok.  You can get the details for the new scripts from CS.

**Review by crspeller — CHANGES_REQUESTED** [review]
Needs to be updated for new load tests. (See Corey's comments)

**Comment by jasonblais** [comment]
@crspeller Which script(s) should I link to instead? I can just remove the settings and link to them.

**Comment by crspeller** [comment]
Will fix after merge.

**Pull request merged**